manifests/plugin/column_view.pp

```diff
@@ -0,0 +1,18 @@
+# = Foreman Column View
+#
+# This class installs the column view plugin and optionally manages the configuration file
+#
+# === Parameters:
+#
+# $columns:: an hash of columns to add to the configuration
+#
```

Comment:
For simple parameter docs I've started to prefer the puppet-strings documentation style
```suggestion
# @summary Install the column view plugin and optionally manage the configuration file
#
# @param columns
#   an hash of columns to add to the configuration
#
```